modules/odc-installing-helm-charts.adoc

```diff
@@ -5,26 +5,19 @@
 [id="odc-installing-helm-charts_{context}"]
 = Installing Helm charts
 
-You can use the *Developer* perspective to create Helm releases from the Helm charts provided in the *Developer Catalog*.
-
-.Prerequisites
-
-* You have logged in to the web console and have switched to the *Developer* perspective.
+You can use either the *Developer* perspective or the CLI to create Helm releases and see them in the Developer perspective of the web console.
```

Comment:
s/releases and/releases and,

Reply:
The clauses around 'and' are dependent on one another, user will see the releases only after she creates them, hence skipping comma.